Blish HUD/GameServices/ArcDps/V2/MessageType.cs

```diff
@@ -1,17 +1,14 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 
 namespace Blish_HUD.GameServices.ArcDps.V2 {
     public enum MessageType {
         // ArcDPS
-        ImGui = 1,
-        CombatEventArea = 2,
+        ImGui            = 1,
+        CombatEventArea  = 2,
         CombatEventLocal = 3,
         // Unofficial Extras
-        UserInfo = 4,
-        ChatMessage = 5,
+        UserInfo         = 4,
+        SquadMessage     = 5,
+        NpcMessage       = 6
```

Comment:
```suggestion
        NpcMessage       = 6,
```

Mainly my codestyle, but it also makes adding stuff better and less error prone